.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "5"
```

Comment:
Testing on node 6 should be better since node 6 is LTS. More about Node.js release schedule can be found [here](https://github.com/nodejs/LTS).